src/watchdog.rs

```diff
@@ -82,6 +129,9 @@ impl WatchdogEnable for SystemWindowWatchdog {
         self.wwdg.cfr.modify(|_, w| w.wdgtb().bits(wdgtb));
```

Comment:
```suggestion
        // TODO: stm32h7 0.14.0 WDGTB is 3 bits
        const WDGTB_MASK: u32 = 0b111 << 11;
        self.wwdg.cfr.modify(|_, w| unsafe { w.bits((r.bits() & !WDGTB_MASK) | (wdgtb << 11) });
```

Reply:
Might as well fix that issue until the PAC is updated, it could be a while.

Reply:
I've now fixed it in the PAC on master. You sure we want this hack in the HAL for the time being?